djangocms_repeater/models.py

```diff
@@ -52,6 +53,9 @@ class Repeater(CMSPlugin):
     repeater_name = models.CharField(max_length=50)
     add_columns = models.BooleanField(default=False)
 
+    def __str__(self):
+        return '%s' % (self.repeater_name)
+
```

Comment:
in the admin if you ever use the models it will display the name or whatever is specified in here rather than how it was showing just ids which was a sea of random numbers